### PR TITLE
HDRP - Clarified LayeredLit UI & doc

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
### Purpose of this PR

Fix for https://fogbugz.unity3d.com/f/cases/1284403/

Added an message box when the selected material is not a Lit or LitTessellation material
image

Updated the [Material To Copy ](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@10.0/manual/Layered-Lit-Shader.html#material-to-copy) section of the doc
Updates the screenshot and adds a section about shader graph materials

new doc

### CODE REVIEW

**Comment by adrien-de-tocqueville** [comment]
Also I think this section of the doc about vertex animation is outdated. Should I remove it ?
https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@10.0/manual/Layered-Lit-Shader.html#vertex-animation

File: com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md [hunk 1]
```diff
@@ -51,17 +51,22 @@ To create a new Layered Lit Material, navigate to your Project's Asset window, r
...
-### Material To Copy
+### Layer List
 
 This section contains a list of the Materials that this Layered Material uses as layers. To assign a Material to a layer, either drag and drop a Material into the property field for that layer, or:
 
 1. Click the radio button on the right of the layer to open the **Select Material** window.
 2. Find the Material you want from the list of Materials in the window and double-click it.
 
-To synchronize the properties between a referenced Material and the Layered Material, press the **Synchronize** button. This copies all of the properties from the referenced Material into the relevant Layered Material layer.
+If the referenced Material gets modified, you can synchronize the properties by pressing the **Reset** button. This copies all of the properties from the referenced Material into the relevant Layered Material layer.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md:61]: If you modify the referenced Material in any way, you can synchronize the properties by pressing the **Reset** button. This copies all of the properties from the referenced Material into the relevant Layered Material layer.

File: com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md [hunk 2]
```diff
@@ -51,17 +51,22 @@ To create a new Layered Lit Material, navigate to your Project's Asset window, r
...
 
 ![](Images/LayeredLit1.png)
 
+If you choose to use a Material made from a Shader Graph as a **Layer Material**, make sure the Reference of the properties match the name of the corresponding properties in the LayeredLit Material.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md:65]: If you assign a Material made from a Shader Graph as a **Layer Material**, make sure the **Reference** of the properties matches the name of the corresponding properties in the LayeredLit Material.

File: com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md [hunk 3]
```diff
@@ -51,17 +51,22 @@ To create a new Layered Lit Material, navigate to your Project's Asset window, r
...
+You can see an example with the **Base Map** in the screenshot below.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Layered-Lit-Shader.md:66]: For an example of what this means, see **_BaseColorMap** in the screenshot below:

**Review by JordanL8 — CHANGES_REQUESTED** [review]
Added a few comments for the docs

**Comment by adrien-de-tocqueville** [comment]
Addressed the requested changes

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**